client/json-rpc/src/async_client/client.rs

```diff
@@ -279,29 +287,43 @@ impl<R: RetryStrategy> Client<R> {
     pub async fn batch_send_with_retry(
         &self,
         requests: &[Request],
```

Comment:
add retry: &RS, to this fn instead of creating new one

Reply:
Sure. Do we need to bump any major semantic version since it'll be a breaking change?

Reply:
Synced offline. No users depend on this so it's fine to update. Done